scripts/drone/vault.star

```diff
@@ -123,6 +123,16 @@ def secrets():
             "infra/data/ci/grafana-release-eng/enterprise2",
             "security_prefix",
         ),
+        vault_secret(
+            "grafana_upstream_repo_main",
+            "infra/data/ci/grafana-release-eng/grafana-enterprise",
+            "upstream-repo-main",
+        ),
+        vault_secret(
```

Comment:
This one doesn't seem to be used anywhere :)

Reply:
Oops, massive c/p :D Thanks!